app/assets/javascripts/common/reports.js

```diff
@@ -1,28 +1,29 @@
+const color = {
```

Comment:
I like `colors` a bit better, since this thing's got a bunch inside.

Reply:
Priyanga mentioned that this might pollute the global space as it is no longer in a local scope.

What are your thoughts?

I will do some reading on imports and global and local scope also

Reply:
Ok, so after doing some digging into the server it looks like all javascript files that are present inside `app/assets/javascripts/common` are imported on every page of the server. 

Meaning all these files become global.

Following Priyanga's thinking there is a risk of variable clashes/overwrites, especially using 'colors' as a name.

I will convert this to a function that return and object of colors for the time being. Ill give it a longer name in an attempt to keep the name relatively unique and hence less likely to suffer clashes/overwrites.